docs/source/dev/dockerfile/dockerfile.rst

```diff
@@ -0,0 +1,12 @@
+Dockerfile
+====================
+
+-  Visualization of the multi-stage Dockerfile
+
+   .. figure:: ../../assets/dev/dockerfile-stages-dependency.png
+      :alt: query
+      :width: 100%
+      :align: center
+
+      Made using https://github.com/patrickhoefler/dockerfilegraph
```

Comment:
Could you include what commands you specific ran in order to generate this so it is simple to update over time?